backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -59,7 +60,19 @@ public UUID createShop(CreateShopRequest createShopRequest) {
 		return newShop.getShopId();
 	}
 
-	public Map<UUID, Shop> getAllShops() {
-		return shopMap;
+	public List<Shop> getAllShops() {
+
+		List<Shop> shopList = new ArrayList<>();
+
+		for (Map.Entry<UUID, Shop> mapOfShop : shopMap.entrySet()) {
+			UUID mapKey = mapOfShop.getKey();
+			Shop mapValue = mapOfShop.getValue();
+
+			if (mapValue.getStatus() == ShopStatus.ACTIVE) {
+				shopList.add(mapValue);
+			}
+		}
```

Comment:
Let's try to use java streamz for making our code more readable here.

Reply:
Yes done